keystore/src/module/keystore.rs

```diff
@@ -199,6 +199,37 @@ impl<'a> MessageWrite for Keystore<'a> {
     }
 }
 
+#[derive(Debug, Default, PartialEq, Clone)]
```

Comment:
If this code generate by protobuf. don't add these code in git, instead of `build.rs`

Reply:
@malefooo